docs/leakscope.md

```diff
@@ -55,9 +70,9 @@ If the following component is defined:
 
     Component.extend({
```

Comment:
break this apart to show the two different ways things "leak".  

1. From outer scope -> into component template.
2. From component template -> into `<content>`